api/envoy/extensions/matching/input_matchers/runtime_fraction/v3/runtime_fraction.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package envoy.extensions.matching.input_matchers.runtime_fraction.v3;
+
+import "envoy/config/core/v3/base.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.matching.input_matchers.runtime_fraction.v3";
+option java_outer_classname = "RuntimeFractionProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/matching/input_matchers/runtime_fraction/v3;runtime_fractionv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
```

Comment:
I think this protodoc to be tagged WIP or hidden or something for now, because the percentage matching part is not actually implemented as described.

Probably also put some TODOs around the code and tests to reflect where that stuff is going to be and what it's going to do.